queries.sql

```diff
@@ -86,4 +86,66 @@ WHERE escape_attempts = 0 AND full_name = 'Dean Winchester';
 
 SELECT full_name , COUNT(animals.name) AS animal_count FROM animals 
 JOIN owners ON animals.owner_id = owners.id
-GROUP BY full_name ORDER BY COUNT(*) DESC LIMIT 1;
\ No newline at end of file
+GROUP BY full_name ORDER BY COUNT(*) DESC LIMIT 1;
+
+
+-- Queries with Link Tables
+
+-- Who was the last animal seen by William Tatcher?
+SELECT animals.name AS animal_name, vets.name AS vet_name, date_visit FROM visits
+JOIN vets ON visits.vets_id = vets.id
+JOIN animals ON visits.animals_id = animals.id
+WHERE vets.name = 'William Tatcher' ORDER BY date_visit DESC LIMIT 1;
+
+-- How many different animals did Stephanie Mendez see?
+SELECT animals.name AS animal_name, vets.name AS vet_name FROM visits
+JOIN vets ON vets.id = visits.vets_id
+JOIN animals ON animals.id = visits.animals_id
+WHERE vets.name = 'Stephanie Mendez';
+
+-- List all vets and their specialties, including vets with no specialties.
+SELECT vets.name AS vet_name , species.name AS species_name FROM vets
+LEFT JOIN specializations ON specializations.vets_id = vets.id
+LEFT JOIN species ON species.id = specializations.species_id;
```

Comment:
- [ ] Some of your queries have wrong results. and the last reviewer has not mentioned it because he asked you to add the data in the visits table. now based on this change, I think you should have updated your screenshots with newer results.
here in this query the result should be something like following
![a](https://github.com/Zuhaib042/vet-clinic-database/assets/96683843/586eaa2a-a16d-47e5-a87b-3e0c33b0e79e)

but your result is this
![w](https://github.com/Zuhaib042/vet-clinic-database/assets/96683843/5e243f4c-fc46-4f5d-822b-4d87f612644b)